src/Features/JsonPatch.SystemTextJson/test/WriteOnceDynamicTestObject.cs

```diff
@@ -0,0 +1,116 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Dynamic;
+
+namespace Microsoft.AspNetCore.JsonPatch.SystemTextJson.Internal;
+
+/// <remarks>
+/// <para>
+/// This class is used specifically to test that JSON patch "replace" operations are functionally equivalent to
+/// "add" and "remove" operations applied sequentially using the same path.
+/// </para>
+/// <para>
+/// This is done by asserting that  no value exists for a particular key before setting its value. To replace the
+/// value for a key, the key must first be removed, and then re-added with the new value.
+/// </para>
+/// <para>
+/// See JsonPatch#110 for further details.
```

Comment:
? See JsonPatch#110 for further details.

Reply:
Interesting, that's an old repo link: https://github.com/aspnet/JsonPatch/issues/110, which addressed while the codebase was in aspnet/jsonpatch repo. I will update that to point to the aspnetcore issue where the original one was moved to: https://github.com/dotnet/aspnetcore/issues/3623